daemon/files.py

```diff
@@ -43,33 +40,13 @@ def workspace_files(
         logger.debug(f'saved uploads to {dest}')
 
 
-def _merge_requirement_file(dest: str, f: UploadFile) -> None:
-    """Merge requirement files
+def is_requirements_txt(filename) -> bool:
```

Comment:
Since this adds support to any requirements.txt file in the format `'.*requirements.*\.txt$'. merging requirements.txt files don't make sense anymore. Hence I have removed that part of the code @jacobowitz